service/src/main/java/uk/gov/hmcts/reform/fpl/model/docmosis/DocmosisCoverDocument.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.fpl.model.docmosis;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+@Builder(builderClassName = "Builder")
+public class DocmosisCoverDocument extends BaseDocmosisDocument {
```

Comment:
I know not directly related to your PR, but what is this a cover document for? Might do us well to be specific here. I believe this is for scanned documents? Maybe a non issue, let me know what you think